src/app/public/modules/angular-tree/angular-tree-wrapper.component.scss

```diff
@@ -70,7 +70,7 @@
   }
 
   .sky-switch-control {
-    margin: 0 $sky-margin 0 0;
+    margin: 0 8px 0 0;
```

Comment:
☝️ The width of our checkboxes doesn't match the width of our caret icons, so unfortunately this couldn't be a nice round number.